packages/next/src/server/use-cache/use-cache-wrapper.ts

```diff
@@ -251,16 +254,16 @@ export function cache(kind: string, id: string, fn: any) {
       // on the server side - instead they need to be recovered as the module
       // references (proxies) again.
       // For now, we'll just use an empty module map.
-      const ssrModuleMap: {
-        [moduleExport: string]: ManifestNode
-      } = {}
+      // const ssrModuleMap: {
+      //   [moduleExport: string]: ManifestNode
+      // } = {}
```

Comment:
We can delete this whole comment now.

Also, we can use this same thing in encryption.ts.